src/test/java/edu/ucsb/cs156/gauchoride/controllers/ShiftControllerTests.java

```diff
@@ -75,6 +77,13 @@ public void logged_out_users_cannot_get_by_id() throws Exception {
                                 .andExpect(status().is(403)); // logged out users can't get by id
         }
 
+        @WithMockUser(roles = { "USER" })
+        @Test
+        public void logged_in_users_can_get_by_id() throws Exception {
+                mockMvc.perform(get("/api/shift/get?id=7"))
+                                .andExpect(status().is(404)); // logged out users can't get by id
```

Comment:
This comment is misleading.  The test doesn't show what this comment says it does.   It's better to have no comment, than a misleading one.